docker-compose.yml

```
@@ -7,24 +7,22 @@ services:
         #   context: .
         #   dockerfile: Dockerfile
         image: borgwarehouse/borgwarehouse
-        # UID:GID must match the user and group ID of the host folders and must be > 1000
-        user: '1001:1001'
+        user: "${UID:?UID variable missing}:${GID:?GID variable missing}"
         ports:
-            - '3000:3000'
-            - '2222:22'
+            - "${WEB_SERVER_PORT:?WEB_SERVER_PORT variable missing}:3000"
+            - "${SSH_SERVER_PORT:?SSH_SERVER_PORT variable missing}:22"
```

```suggestion
            - "${SSH_SERVER_PORT:?SSH_SERVER_PORT variable missing}:2222"
```
Port 22 is taken by default by the host's SSH service. It's a good idea to specify another default port, even if you're free to set it as you wish, to avoid a startup error if the port hasn't been changed.